### PR TITLE
Moar fixes before 2.4.0 release

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

File: apps/desktop-wallet/src/components/AmountsOverviewPanel.tsx [hunk 1]
```diff
@@ -65,7 +66,11 @@ const AmountsOverviewPanel = ({
             <BalancesColumn>
               <Today>{!hoveredDataPointDate ? t('Value today') : `${hoveredDataPointDate} (${t('ALPH only')})`}</Today>
 
-              <WalletWorth overrideWorth={hoveredDataPointWorth} />
+              {singleAddress ? (
+                <AddressWorth overrideWorth={hoveredDataPointWorth} addressHash={addressHash} />
+              ) : (
+                <WalletWorth overrideWorth={hoveredDataPointWorth} />
+              )}
```
Comment (nop33) [inline apps/desktop-wallet/src/components/AmountsOverviewPanel.tsx:73]: This was one of the bugs I discovered yesterday

File: apps/desktop-wallet/src/features/walletConnect/walletConnectContext.tsx [hunk 2]
```diff
@@ -603,6 +603,7 @@ export const WalletConnectContextProvider: FC = ({ children }) => {
   useEffect(() => {
     if (walletLockedBeforeProcessingWCRequest && sessionRequestEvent && addresses.length > 0) {
       onSessionRequest(sessionRequestEvent)
+      setWalletLockedBeforeProcessingWCRequest(false)
```
Comment (nop33) [inline apps/desktop-wallet/src/features/walletConnect/walletConnectContext.tsx:606]: and this was the second

File: apps/desktop-wallet/src/features/send/sendModals/transfer/TransferSendModal.tsx [hunk 3]
```diff
@@ -101,7 +101,7 @@ export const handleTransferSend = async (transactionData: TransferTxData, contex
         )
       }
 
-      posthog.capture('Swept address assets')
+      posthog.capture('Swept address assets', { from: 'button' })
```
Comment (mvaivre) [inline apps/desktop-wallet/src/features/send/sendModals/transfer/TransferSendModal.tsx:104]: 👍

File: apps/desktop-wallet/src/features/walletConnect/walletConnectContext.tsx (hunk 2 above)
Comment (mvaivre) [reply apps/desktop-wallet/src/features/walletConnect/walletConnectContext.tsx:606]: well done! 👏

**Comment by nop33** [comment]
You left a "comment" review but with no action points. Does that mean it's an approval? :P

**Review by mvaivre — APPROVED** [review]
LGTM!

**Pull request merged**